nix/cells/automation/pipelines.nix

```diff
@@ -39,10 +44,46 @@ in
     nomad.resources.cpu = 10000;
   };
 
+  benchmark = { config, ... }:
+    let
+      # Facts here correspond to inputs created by the benchmark action in the cloud cell
+      facts = config.actionRun.facts;
+      commentFact = facts.${cloud.library.actions.benchmark.commentInput}.value.github_body;
+      prFact = facts.${cloud.library.actions.benchmark.prInput}.value.github_body;
+
+      prRevision = prFact.pull_request.head.sha;
+
+      runner = cell.library.plutus-benchmark-runner {
+        PR_NUMBER = toString prFact.pull_request.number;
+        PR_COMMIT_SHA = prRevision;
+        BENCHMARK_NAME = lib.removePrefix "/benchmark " commentFact.comment.body;
+        GITHUB_TOKEN = "/secrets/cicero/github/token";
+      };
+    in
+    {
+      # Not importing common, github preset is not needed here
+      preset.nix.enable = true;
+      preset.git.clone = {
+        enable = true;
+        remote = "https://github.com/input-output-hk/plutus";
+        ref = prRevision;
+      };
+      command.text = "${runner}/bin/plutus-benchmark-runner";
+      nomad.templates = [
+        {
+          destination = "/secrets/cicero/github/token";
+          data = ''{{with secret "kv/data/cicero/github"}}{{.Data.data.token}}{{end}}'';
+        }
+      ];
+
+      memory = 1024 * 8;
+      nomad.resources.cpu = 10000;
```

Comment:
Are there default values? What does 8G, and 10000 mean? How do they differ from the default?

Reply:
[Default](https://developer.hashicorp.com/nomad/docs/job-specification/resources#resources-parameters) is 300 MB, 100 MHz